Cargo.toml

```diff
@@ -12,6 +12,6 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 thiserror = "1"
-serde = {version = "1.0.123", optional = true, default-features = false}
```

Comment:
serde = {version = "1.0.123", optional = true, features = ["derive"] } should fix things